src/components/EditProfileForm.tsx

```diff
@@ -108,14 +109,33 @@ const EditProfileForm = () => {
     (store) => store.user.authUser._id
   );
 
+  const [avatarImage, setAvatarImage] = useState<FormData>();
+  const [bannerImage, setBannerImage] = useState<FormData>();
+
+  const uploadedAvatar = (avatarImageProps: FormData) => {
+    setAvatarImage(avatarImageProps);
+  };
+  const uploadedBanner = (bannerImageProps: FormData) => {
+    setBannerImage(bannerImageProps);
+  };
+
   const submitForm = (data: IFormInput) => {
     dispatch(userActions.editProfileUser({ ...data, userId }));
+    if (avatarImage) {
+      dispatch(userActions.addAvatarAsync({ avatarImage, userId }));
+    }
+    if (bannerImage) {
+      dispatch(userActions.addBannerAsync({ bannerImage, userId }));
```

Comment:
Until someone has implemented adding a banner to the backend, you don't need to make that AsyncThunk call on the submit form.
```suggestion
      // dispatch(userActions.addBannerAsync({ bannerImage, userId }));
```

Reply:
I suppose that on the backend the banner will work like an avatar, so I have already organized a banner replacement on the frontend. So when the banner appears on the backend, nothing needs to be done on the frontend

Reply:
@KirillSerg unfortunately, it doesn't work(

Reply:
done